v-next/hardhat-build-system/src/internal/solidity/compiler/adm-zip.d.ts

```diff
@@ -0,0 +1 @@
+declare module "adm-zip";
```

Comment:
This appeared to be necessary once we moved to `v-next`. I will look again at why we are not picking up the types.